composables/settings/storage.ts

```diff
@@ -43,5 +43,5 @@ export function getPreferences<T extends keyof PreferencesSettings>(userSettings
 
 export function togglePreferences(key: keyof PreferencesSettings) {
   const flag = usePreferences(key)
-  flag.value = !flag.value
+  return flag.value = !flag.value
```

Comment:
Added this explicit return so that I could use it in the `toggleOptOut()` function later, will call it out

Reply:
Returning a variable assignment expression is a new one to me. Buut if it works, it works 👍

Reply:
I'm such a rubyist at heart. In Ruby, every method (function) returns the evaluated result of the last line that is executed. And it was my first modern language, so I often forget that javascript doesn't do what I think it should do sometimes, but really I'm just remembering back to my ruby days.